VS/VehicleSignals.rdf

```diff
@@ -2290,6 +2290,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;Intake-AirTemperatureSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Intake air temperature sensor</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.